lms/templates/instructor/edx_ace/addbetatester/email/body.html

```diff
@@ -7,34 +7,44 @@
     <tr>
         <td>
             <h1>
+                {% filter force_escape %}
                 {% blocktrans %}
                     You have been invited to be a beta tester for {{ course_name }} at {{ site_name }}
                 {% endblocktrans %}
+                {% endfilter %}
             </h1>
 
             <p style="color: rgba(0,0,0,.75);">
+                {% filter force_escape %}
                 {% blocktrans %}The invitation has been sent by a member of the course staff.{% endblocktrans %}
+                {% endfilter %}
                 <br />
             </p>
 
             {% if auto_enroll %}
                 <p style="color: rgba(0,0,0,.75);">
+                    {% filter force_escape %}
                     {% blocktrans %}To start accessing course materials, please visit:{% endblocktrans %}
+                    {% endfilter %}
                     <br />
                 </p>
 
                 {% include "ace_common/edx_ace/common/return_to_course_cta.html" with course_cta_text=display_name|default:course.display_name_with_default course_cta_url=course_url %}
             {% elif course_about_url is not None %}
                 <p style="color: rgba(0,0,0,.75);">
+                    {% filter force_escape %}
                     {% blocktrans %}To enroll in this course and begin the beta test:{% endblocktrans %}
+                    {% endfilter %}
                     <br />
                 </p>
 
                 {% blocktrans asvar course_cta_text %}Visit {{ course_name }}{% endblocktrans %}
                 {% include "ace_common/edx_ace/common/return_to_course_cta.html" with course_cta_text=course_cta_text course_cta_url=course_about_url %}
```

Comment:
1. I'm unclear if this is safe, or if it needs the same `|safe` or some other feature like `interpolate_html`?
2. Once you understand what is needed, can the linter catch this?

Reply:
For now iam not doing anything for them. My understanding is that each argument/params would be escaped by the called template and its responsibility to do that.  So if return_to_course_cta.html is using this param then we must be escaping the param in that template.

Reply:
Sounds good.